packages/terra-clinical-result/docs/README.md

```diff
@@ -0,0 +1,169 @@
+# Terra Clinical Result
+
+The Terra Clinical Result package is a collection of standardized views for presenting clinical results documented to a Patient's Medical Chart, 
```

Comment:
There is a 500 error when loading the [doc site](https://terra-clinical-deployed-pr-574.herokuapp.com/components/terra-clinical-result/clinical-result/clinical-result). May need to add [these lines](https://github.com/cerner/terra-clinical/blame/master/packages/terra-clinical-header/README.md#L4-L5) for the version and build status badges.